ios/Sources/AutoLayoutViewComponentView.mm

```diff
@@ -0,0 +1,80 @@
+#ifdef RN_FABRIC_ENABLED
```

Comment:
Change all of the `RN_FABRIC_ENABLED` to  `RCT_NEW_ARCH_ENABLED`. Otherwise it won't work correctly since we don't add such flag on new arch.

Reply:
seems like an easy change, anyway can I contribute?

afaik I'd have to clone the branch then merge my edit to it for it be available in this PR, is this correct or is there a better/faster way?

Reply:
https://github.com/markrickert/flash-list/pull/1 @markrickert

Reply:
@WoLewicki I made this change to `RCT_NEW_ARCH_ENABLED`